core/src/modules/telemetry.ts

```diff
@@ -40,7 +40,7 @@ export namespace Telemetry {
         });
       } catch (newError) {}
 
-      throw error;
+      log(`[ telemetry ] ${error}`, "error");
```

Comment:
this might be good for `grouparoo run` but mahybe it should retry in the `grouparoo start` background case?

Reply:
👍 The tasks weren't retryable... they are now!